engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -337,6 +355,22 @@ class EngineJobExecutionActor(replyTo: ActorRef,
     goto(RunningJob) using data
   }
 
+  private def handleCacheInvalidatedResponse(response: CallCacheInvalidatedResponse, data: ResponsePendingData) = {
+    response match {
+      case CallCacheInvalidatedFailure(failure) => log.error(failure, "Failed to invalidate cache entry for job: {}", jobDescriptorKey)
```

Comment:
I think it's fine if it doesn't get retried. Like you're saying, it seems like a retry for failed transactions is something more global and thus might be worth implementing in general instead of for this specific case.